### PR TITLE
Improve CONTRIBUTING file, add issue & PR template

### PR DESCRIPTION
**Pull request by KILLEliteMaste**
Below you will see how the PR template is rendered.

---

## Checklist
- [ ] I have tested this PR[^1].
- [ ] I have read the [contributing guidelines](./CONTRIBUTING.md).

## Changelog
- 

### Breaking Changes

Closes Issue: NaN

[^1]: At least started a running bot instance with your changes and triggered an event so your changed code gets executed.

### CODE REVIEW

**Comment by RealYusufIsmail** [comment]
what just thinking about this

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+If there are breaking changes list them separately.
+-->
+## Changelog
+- 
+
+### Breaking Changes
+
+
+<!-- Replace "NaN" with an issue number if this is a response to an issue -->
+Closes Issue: NaN
```
Comment (RealYusufIsmail) [inline .github/PULL_REQUEST_TEMPLATE.md:27]:
```suggestion
closes : NaN
```
I would change it to this, as Closes Issue does not really register with Github so it does not actually show it in Development section but closes :  works

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+
+<!-- 
+Write down the changes this PR introduces. 
+If there are breaking changes list them separately.
+-->
+## Changelog
+- 
+
+### Breaking Changes
```
Comment (RealYusufIsmail) [inline .github/PULL_REQUEST_TEMPLATE.md:23]:
```suggestion
### Changes

- [ ] Existing code
- [ ] New feature

<!-- 
Write down the changes this PR introduces. 
If there are breaking changes list them separately.
-->
## Changelog
- 

### Breaking Changes
```

Comment (RealYusufIsmail) [reply .github/PULL_REQUEST_TEMPLATE.md:23]: up to you

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<!-- 
+A brief description of what this PR is about if the title is not sufficient
+-->
+## Description
+
+
```
Comment (RealYusufIsmail) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: In my opinion this should go at the bottom, it means it less cluttered

**Review by RealYusufIsmail — CHANGES_REQUESTED** [review]

**Comment by RealYusufIsmail** [comment]
Also do you mind adding an issue template, i.e if its an issue or a feature request.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (KILLEliteMaste) [reply .github/PULL_REQUEST_TEMPLATE.md:23]: There are arguments against this approach https://github.com/Javacord/Javacord/issues/767#issuecomment-856255155 which I agree with

Comment (RealYusufIsmail) [reply .github/PULL_REQUEST_TEMPLATE.md:23]: ok

**Comment by KILLEliteMaste** [comment]
Bug Report issue form:
github com_KILLEliteMaste_Javacord_blob_improvement_contributing_ github_ISSUE_TEMPLATE_bug_report yml

Feature request template form:
github com_KILLEliteMaste_Javacord_blob_improvement_contributing_ github_ISSUE_TEMPLATE_bug_report yml (3)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (RealYusufIsmail) [reply .github/PULL_REQUEST_TEMPLATE.md:27]: Why was this ignored

**Review by RealYusufIsmail — CHANGES_REQUESTED** [review]
Other than LGTM

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+description: Suggest an idea for this project
+labels: [":sparkles: feature request"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Before You Start...**
+
+        This form is only for submitting bug reports. If you have a usage question
+        or are unsure if this is really a bug, make sure to:
```
Comment (Saladoc) [inline .github/ISSUE_TEMPLATE/feature_request.yml:11]:
```suggestion
        This form is only for submitting feature requests. If you have a usage question
        or are unsure if the feature you propose does exist already, make sure to:
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        - Ask on [Discord Chat](https://discord.gg/javacord)
+        
+         Also try to search for your issue - another user may have already requested something similar!
+  - type: textarea
+    id: feature-description
+    attributes:
+      label: What feature are you proposing?
+      placeholder: Explain your feature if the title is not precise enough.
+    validations:
+      required: false
```
Comment (Saladoc) [inline .github/ISSUE_TEMPLATE/feature_request.yml:24]: The placeholder text is at odds with the validations. The placeholder text suggests this being optional, the validation enforces it. Either one must change to be consistent.

**Review by Saladoc — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 5 above)
Comment (KILLEliteMaste) [reply .github/ISSUE_TEMPLATE/feature_request.yml:24]: I am not sure what you mean. The validation rule is set to **not** required.

Comment (Saladoc) [reply .github/ISSUE_TEMPLATE/feature_request.yml:24]: So I either had a display error or a mixup here. That's strange.

**Pull request merged**